django/db/backends/sqlite3/base.py

```diff
@@ -49,6 +50,17 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def list_aggregate(function):
+    """
+    Returns an aggregate class that accumulates values in a list and applies
+    the provided function to the data.
+    """
+    return type('ListAggregate', (list,), {
+        'step': list.append,
+        'finalize': lambda self: function(self),
```

Comment:
Is the `lambda` required here?

Reply:
Ah, no - missed cleaning that up... :+1: